### ISSUE TITLE
[QUESTION] Pretraining dinov2_vitb14 on 200k Custom Images — Loss Curve Evaluation and Optimization Suggestions

### ISSUE THREAD

**Issue by changcongxun** [lightly-ai/lightly-train#481]
### 🤔 What’s your question?
Hi, I pretrained dinov2_vitb14 on a custom dataset containing 200,000 RGB images.
Here are my training settings:
    Total epochs: 3000
    Batch size: 512
    GPUs: 8 × NVIDIA A800-SXM4-80GB
Below is the loss curve recorded by TensorBoard.

<img width="1236" height="659" alt="Image" src="https://github.com/user-attachments/assets/abc9f2c2-adb9-4ae1-ac2f-4a4d3db3fa6b" />
Could you please help me analyze whether the training process and the loss behavior look correct?
Are there any potential improvements or optimization suggestions for better convergence or performance?

Thanks!
### 📌 Related code or context (if any)

<img width="885" height="296" alt="Image" src="https://github.com/user-attachments/assets/84088f9b-c786-4be9-943a-959d4ef2b799" />

**Comment by stegmuel** [comment]
Hi,
I would say that it did look decent until ~250k steps, afterwards things get a bit suspicious.
The default number of training steps is 125k. Did you try with the default setting first?
In general we found that using a number of steps close to the default helps a lot with stability. The capacity of the training  can also be modulated with a larger batch size or larger models.
